packages/amplify_authenticator/lib/src/views/confirm_signin_viewmodel.dart

```diff
@@ -151,7 +141,7 @@ class ConfirmSignInViewModel extends BaseViewModel {
 
     _authBloc.authEvent.add(AuthConfirmSignIn(confirm));
     await Future.any([
-      _authBloc.exceptions.first,
+      _authBloc.exceptions!.first,
       _authBloc.stream.first,
```

Comment:
We need to update this in all the VMs since each event can trigger multiple state changes now 

(swap `Authenticated` for whichever state we're waiting for)
```dart
_authBloc.stream.firstWhere((state) => state is Authenticated)
```